### DISCUSSION TITLE
Hue app has mandatory update for hue bridge

### DISCUSSION THREAD

**Discussion by ryknow27** [hass-emulated-hue/core#333] [Q&A]
Is there a way to bypass the mandatory hue bridge update? I am not able to add devices to the virtual hub because of the required update.

**Comment by ryknow27** [comment]
Disregard, I just saw this same issue listed in the issues section. Sorry

**Reply by TonCoronel** [reply]
Does this mean that we simply cant use this add-on at the moment? I run into the same issue

**Reply by ryknow27** [reply]
I wasn't able to get it working since then and haven't tried again.

**Reply by Al3eRtO** [reply]
Same issue, news?

**Comment by ofirda** [comment]
Same here - any resolution or alternative?

**Comment by kimiseppanen** [comment]
Any updates?
